test/support/protocol_utils.rb

```diff
@@ -347,10 +347,16 @@ def attach_to_dap_server
     HOST = '127.0.0.1'
 
     def attach_to_cdp_server
+      Timeout.timeout(TIMEOUT_SEC) do
```

Comment:
Could you align this waiting logic with DAP part?

Reply:
Here: https://github.com/ruby/debug/blob/master/test/support/protocol_utils.rb#L239

Reply:
Here: https://github.com/ruby/debug/blob/master/test/support/protocol_utils.rb#L239

Reply:
Sure, I've updated it.